apps/remix-ide/src/app/tabs/theme-module.js

```diff
@@ -33,7 +78,6 @@ export class ThemeModule extends Plugin {
       config: registry.get('config').api
     }
     this.themes = themes.reduce((acc, theme) => {
-      theme.url = window.location.origin + window.location.pathname + theme.url
```

Comment:
@yann300

Reply:
I think we still need that one

Reply:
@igor-lid

Reply:
I remove this line because I thought the CDN link is going to be used to load the theme css. e.g. ` url: 'https://res.cloudinary.com/lianahus/raw/upload/v1597918237/remix-themes/PR365/remix-dark_tvx1s2.css'`

Or is it better to rollback to previous logic where the theme css is imported from the assets folder? e.g.  `url: 'assets/css/themes/remix-dark_tvx1s2.css'`